### ISSUE TITLE
Please Upscale with efficiency

### ISSUE THREAD

**Issue by blueps5controller** [TheKiritoPlayer20/MoreOresInONE#2]
I've spent like the last 30 minutes in a creative world testing your pickaxes and I've come to the conclusion that while they do mine faster with efficiency, it isn't noticeable enough to be like "wow! This was worth all the time it took to get everything!" Y'know? I'm not trying to rip on you since you made a pretty nice mod! But if a Amethyst Pickaxe that's enchanted with efficiency 5 isn't fast enough to be distinguishable from a Diamond Pickaxe enchanted with efficiency 5 then why bother y'know?
So basically...what I'm trying to say is please make the pickaxes faster! They just aren't worth the trouble of going through to get since they don't FEEL faster than diamond and if you can't notice it then like I said before why bother?
Thank You, Much Love
- Rosenhiem

**Comment by realKG20** [comment]
Minecraft issue with the Enchantments. You can also try the same thing with Gold Pickaxes and Diamond Pickaxes as far as i know. Still gonna look into it later today

**Comment by realKG20** [comment]
I tested it myself and i come to the conclusion, that while unenchanted the Amethyst Pickaxe is a bit faster, that they are having the same mining speed on Efficiency 5 even after buffing the raw Efficiency of the Amethyst Pickaxe.

Times:
1:40 Amethyst Pickaxe unenchanted
1:30 Amethyst Pickaxe unenchanted (Buffed)
2:00 Diamant   Pickaxe unenchanted

1:20 Amethyst Pickaxe enchanted(Efficiency 5)
1:20 Diamant   Pickaxe enchanted(Efficiency 5)

Tested in a 10x2x10 area.

**Comment by realKG20** [comment]
I think i have the right amount of speed now, you can insta mine stone blocks with Efficiency 5 but only with Efficiency 5. The Pickaxe would have the double the base efficiency of the amethyst pickaxe which is out right now.

**Comment by realKG20** [comment]
Updated in version 1.4
